graph/src/data/subgraph/mod.rs

```diff
@@ -854,6 +892,8 @@ pub struct BaseSubgraphManifest<S, D, T> {
     pub repository: Option<String>,
     pub schema: S,
     pub data_sources: Vec<D>,
+    #[serde(rename = "graft")]
+    pub base: Option<Graft>,
```

Comment:
Why not have the field name match the serialized name?

Reply:
I am sure I had a very good reason - just changed it to `graft`